pkg/rpc/client/rpc_test.go

```diff
@@ -806,6 +807,46 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 			},
 		},
 	},
+	"invokecontractverify": {
```

Comment:
Can we add this test without prebuilt response to `server/client_test.go` instead?